lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -236,6 +257,14 @@ class VersionSectionSelector extends Component<SelectorProps> {
               <Icon type='exclamation-triangle' /> Validation issues {this._renderIssuesLabel()}
             </ListGroupItem>
           </LinkContainer>
+          <LinkContainer
+            to={`${versionPath}/transformations`}
+            active={versionSection === 'transformations'}
```

Comment:
Sort props.

Reply:
They're all wrong nearby this. Should I fix all of them?

Reply:
Sure